index.html

```diff
@@ -22,25 +23,195 @@
+                        <li class="css">css</li>
+                        <li class="javascript">javaScript</li>
+                    </ul>
+                    <button class="btn" type="button">See Project</button>
+                </div>
+            </div>
+
+            <!-- ******Project 3****** -->
+            <div class="Project">
+                <img class="pic1" src="./Images/snap3.svg" alt="Second-Project">
+                <div class="body">
+                    <h2 class="title">Tonic</h2>
+                    <ul class="Can-2015">
+                        <li class="canopy">CANOPY</li>
+                        <li><img class=counter src="./Images/Counter.png" alt="counter"></li>
+                        <li class="back">Back End Dev</li>
+                        <li><img class=counter src="./Images/Counter.png" alt="counter"></li>
+                        <li class="year">2015</li>
+                    </ul>
+                    <p class="info">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required.
+                    </p>
+                    <ul class="lang">
+                        <li class="html">html</li>
+                        <li class="css">css</li>
+                        <li class="javascript">javaScript</li>
+                    </ul>
+                    <button class="btn" type="button">See Project</button>
+                </div>
+            </div>
+
+
+            <!-- ******Project 4******  -->
+            <div class="Project">
+                <img class="pic1" src="./Images/snap4.svg" alt="First-Project">
+                <div class="body">
+                    <h2 class="title">Multi-Post Stories</h2>
+                    <ul class="Can-2015">
+                        <li class="canopy">CANOPY</li>
+                        <li><img class=counter src="./Images/Counter.png" alt="counter"></li>
+                        <li class="back">Back End Dev</li>
+                        <li><img class=counter src="./Images/Counter.png" alt="counter"></li>
+                        <li class="year">2015</li>
+                    </ul>
+                    <p class="info">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required.
+                    </p>
+                    <ul class="lang">
+                        <li class="html">html</li>
+                        <li class="css">css</li>
+                        <li class="javascript">javaScript</li>
+                    </ul>
+                    <button class="btn" type="button">See Project</button>
+                </div>
+            </div>
+
+
+        </section>
+        <!-- /*About myself ----------------------------section*/ -->
+        <section class="myself">
+            <div class="Top-part">
+                <h3>About Myself</h3>
```

Comment:
- [ ] As mentioned before please :pray: try to reduce the spaces between some elements inside your project to match with the Figma design :+1: 
- [ ] Also I have noticed that the title of your about me section is not matching with figma design as you can see below 

**Your design**

![Screenshot from 2023-01-14 08-09-36](https://user-images.githubusercontent.com/105117832/212473270-7fb82c28-0a9f-4058-9b72-860a103f051a.png)

**Figma design**

![Screenshot from 2023-01-14 08-09-53](https://user-images.githubusercontent.com/105117832/212473284-e9817417-c8b5-458f-a98d-705473c63b56.png)